src/main/java/org/mitre/synthea/export/FhirStu3.java

```diff
@@ -130,6 +131,9 @@
   @SuppressWarnings("rawtypes")
   private static final Map languageLookup = loadLanguageLookup();
 
+  @SuppressWarnings("rawtypes")
+  private static final Map codeSystemLookup = loadLookupTable();
```

Comment:
This is ok but it's going to need to be used by both the FHIR 2 and 3 exporters, and potentially more, so it would be good to put it into a common place; we don't want to load the same file into memory twice. I'd propose either ExportHelper or Utilities